SublimeLinter.sublime-settings

```diff
@@ -55,11 +55,11 @@
     // - save: only when a file is saved
     "lint_mode": "background",
 
-    // Linter specific settings.
+    // Linter-specific settings.
     // More info: http://www.sublimelinter.com/en/stable/linter_settings.html
-    // Linter specific settings except for 'styles' can also be changed
+    // Linter-specific settings except for 'styles' can also be changed
     // in sublime-project settings.
-    // What settings are available is documented in the readme of the
+    // What settings are available are documented in the readme of the
```

Comment:
really?  Isn't that here the short form of "It is documented ...".  Not "other possible settings are documented".

Reply:
The “it” in this case is plural (“linter-specific settings”).

Reply:
Ok, I'm not a native speaker.  I thought "Additional settings _are_ documented in the readme." but "Which settings are valid here _is_ documented in the readme." because it is not the subject.  I can transform the sentence to "What settings are allowed here *I* wrote down over there." or typically flip "*It* is documented in the readme what/which settings are further available".   LOL, but what do I know. :grin: